scripts/build.js

```diff
@@ -0,0 +1,106 @@
+const { execSync } = require("child_process")
+const { mkdirSync, renameSync, rmSync, existsSync } = require('fs')
+const path = require('path')
+const crypto = require('crypto')
+
+const fs = require('fs')
+const https = require('https')
+
+function execPrint(cmd) {
+    console.log(execSync(cmd).toString())
+}
+
+
+async function run(name, scheme, tauname) {
+    const root = process.cwd()
+    const tauFile = path.join(root, 'tau', tauname)
+    const build = path.join(root, 'build')
+    const circuitFile = path.join(root, 'circuits', name+'.circom')
+    const setup = path.join(build, 'setup', scheme, name)
+    const dest = path.join(root, 'compiled', name)
+    if (!existsSync(tauFile)) {
+        const tauURL = `https://hermez.s3-eu-west-1.amazonaws.com/${tauname}`
+        mkdirSync(path.join(root, 'tau'), {recursive: true})
+        console.log(`Downloading ${tauURL} and saving it to ${tauFile}`)
+        try {
+            await downloadFile(tauURL, tauFile)
+        } catch (e) {
+            console.error(e)
+            process.exit()
```

Comment:
And I'm thinking process should exit with a error code, e.g. 
```suggestion
            process.exit(1)
```
This would be useful for users (e.g. CI) to detect errors